documentation/general/design/dash-holistic-view.md

```diff
@@ -0,0 +1,618 @@
+traffic.
+
+Each DASH NIC/DPU Will run a version of SONiC that exposes its own gNMI endpoint
+for SDN Control. This endpoint is reachable in band through the "front-panel"
+DPU traffic ports via L3 routing. In other words, the SDN controller can reach
+the DPU management endpoints over the ToR-to-DPU fabric links.
+
+In some cases, DPUs might provide separate management Ethernet ports, or PCIe
+netdevs which can be used for control purposes, in accordance with deployment
+and security needs.
+
+#### High Level Architecture
+
+![dash-high-level-appliance](images/hld/dash-high-level-appliance.svg)
+
+<figcaption><i>Figure 5.1 - Appliance high level</i></figcaption><br/><br/>
+
+#### Low level architecture
+
+![dash-appliance-architecture](images/hld/dash-appliance-architecture.svg)
+
+<figcaption><i>Figure 5.2 - Appliance low level</i></figcaption><br/><br/>
+
+### DASH smart switch
+
+A DASH **smart switch** is a merging of a datacenter switch and one or more DPUs
+into an integrated device. The "front-panel" network interfaces of the DPU(s)
+are wired directly into the switching fabric instead of being presented
+externally, saving cabling, electronics, space and power. There can also be some
+consolidation of software stacks, for example see SONiC Multi-ASIC for how this
+is accomplished in standard SONiC multi-ASIC devices.
+
+#### High level architecture
+
+![dash-high-level-smart-switch](images/hld/dash-high-level-smart-switch.svg)
+
+<figcaption><i>Figure 6.1 - Smart switch high level</i></figcaption><br/><br/>
+
+#### Low level architecture
+
+![dash-smart-switch-architecture](images/hld/dash-smart-switch-architecture.svg)
+
+<figcaption><i>Figure 6.2 - Smart switch low level</i></figcaption><br/><br/>
+
+## Example of network representation 
+
+> [!NOTE] TBD. Introduction needed.
+### Deployment
+
+The following figure is a simplified representation of DASH deployment in a
+datacenter.
+
+![dash-simplified-physical-deployment-example](images/dash-simplified-physical-deployment-example.svg)
+
+<figcaption><i>Figure 7 - DASH simplified deployment</i></figcaption><br/>
+
+## API
+
+> [!NOTE] TBD  - Describe in general terms the intent and where it *fits* in
+> the *big picture*. 
+
+### Switch Abstraction Interface (SAI) API
+
+The Switch Abstraction Interface (SAI) is a common API that is supported by many
+switch ASIC technology suppliers. SONiC uses the API to work across multiple
+ASIC platforms naturally. DASH uses a combination of traditional SAI headers and
+new DASH pipeline-specific headers. Technology suppliers must implement this
+interface for their DASH devices. This interface is the primary integration
+point of DASH devices and the SONiC stack. It will be rigorously tested for
+performance and conformance. See DASH Testing documentation.
+
+See related code at this location
+[SAI](https://github.com/Azure/DASH/tree/main/SAI). See also:
+
+- [Overlay](https://github.com/Azure/DASH/tree/main/SAI/overlay)
```

Comment:
These directories no longer exist. The SAI header files for overlay are generated from the P4 code but not stored as source in GitHub. The SAI headers for Underlay are a filtered subset derived from standard OCP SAI headers. [Check with Marian P.]

Reply:
@chrispsommers Fixed the content based on your comment, for now. Please verify.